servicemesh-install/topics/servicemesh-istio-comparison.adoc

```diff
@@ -0,0 +1,45 @@
+[[servicemesh-istio-comparison]]
+= Comparing {ProductName} and upstream Istio community installations
+An installation of {ProductName} differs from upstream Istio community installations in multiple ways. The modifications to {ProductName} are sometimes necessary to resolve issues, provide additional features, or to handle differences when deploying on OpenShift.
+
+The current release of {ProductName} differs from the current upstream Istio community release in the following ways:
+
+== Automatic injection
+The upstream Istio community installation automatically injects the sidecar to namespaces you have labeled.
+
+{ProductName} does not automatically inject the sidecar to any namespaces, but requires you to specify sidecar injection in the https://docs.openshift.com/container-platform/3.11/servicemesh-install/servicemesh-install.html#updating-master-configuration[master configuration].
+
+== Role Based Access Control features
+Role Based Access Control (RBAC) provides a mechanism you can use to control access to a service. You can identify subjects by username or by specifying a set of properties and apply access controls accordingly.
+
+The upstream Istio community installation includes options to perform exact matches, or check for a header containing a specific prefix or suffix.
```

Comment:
Sorry, there should also be a wildcard match in this list

exact, wildcard, prefix and suffix.